src/test/mochitest/head.js

```diff
@@ -770,12 +773,18 @@ function findAllElements(dbg, elementName, ...args) {
  * @return {Promise}
  * @static
  */
-function clickElement(dbg, elementName, ...args) {
+async function clickElement(dbg, elementName, ...args) {
   const selector = getSelector(elementName, ...args);
+  await waitForElement(dbg, selector);
```

Comment:
`waitForElement` could directly return the element we wait for, it would save us the extra `findElement` on the next line.